org.jjazzlab.JJazzLab.yml

```diff
@@ -0,0 +1,97 @@
+id: org.jjazzlab.JJazzLab
```

Comment:
https://jjazzlab.org/ is an OVH Site construction page

Reply:
The current website jjazzlab.com is in the process of being moved to jjazzlab.org. It therefore seems more sensible to use the new domain. Does that make sense?

Reply:
That's a bit problematic, since there's no way to verify ownership. If for some reason the transfer did not happen, the application ends up with the wrong id preventing future verification etc.

Reply:
The main website has been migrated to jjazzlab.org.